.github/workflows/test.yml

```diff
@@ -0,0 +1,105 @@
+name: Terraform Testing
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+    branches: [master]
+    paths:
+      - "modules/**"
+
+jobs:
+  check:
+    name: Check
+    runs-on: ubuntu-latest
+    outputs:
+      packages: ${{ env.package }}
```

Comment:
Does this value available in this step? I believe it only works because it is not the first run